sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/impl/ConfigurationCondition.java

```diff
@@ -87,4 +87,9 @@ public int hashCode() {
     public int compareTo(ConfigurationCondition o) {
         return this.typeName.compareTo(o.typeName);
     }
```

Comment:
Maybe `objectReachable()` should really be named `alwaysTrue()` for easier comprehension of code that registers types, proxies, etc.